48_positional_encoding_timeseries/python/examples/time_series.py

```
@@ -0,0 +1,158 @@
+
+    # Crypto market (24/7)
+    print("\n3a. Cryptocurrency (24/7)")
+    session_crypto = MarketSessionEncoding(d_model, market_type='crypto')
+
+    # Generate hourly timestamps for a day
+    day_timestamps = torch.tensor([
+        int((base_time + timedelta(hours=i)).timestamp())
+        for i in range(24)
+    ]).unsqueeze(0)
+
+    session_encoded = session_crypto(day_timestamps)
```

Ruff (0.14.13)

Local variable `session_encoded` is assigned to but never used

Remove assignment to unused variable `session_encoded`

(F841)